Modules/QuestieNameplate.lua

```diff
@@ -5,14 +5,9 @@ local npFrames = {};
 local npUnusedFrames = {};
 local npFramesCount = 0;
 
-QuestieNameplate.GlobalFrame = nil;
-
 -- Initializer
 function QuestieNameplate:Initialize()
-    if QuestieNameplate.GlobalFrame == nil then
-        QuestieNameplate.GlobalFrame = CreateFrame("Frame");
-        QuestieNameplate.GlobalFrame:SetScript("OnUpdate", QuestieNameplate.UpdateNameplate);
-    end
+	Questie:RegisterMessage("QUESITE_ICON_ON_UPDATE",QuestieNameplate.UpdateNameplate)
```

Comment:
```suggestion
	Questie:RegisterMessage("QUESTIE_ICON_ON_UPDATE",QuestieNameplate.UpdateNameplate)
```

Reply:
No GlobalFrame in the code anymore. this needs to be removed.